inst/rmarkdown/templates/vetiver_dashboard/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,91 @@
+---
+title: "Monitor model"
+output: 
+  vetiver::vetiver_dashboard:
+    board: !expr pins::board_rsconnect()
+    name: 'julia.silge/hotel_rf'
+    storyboard: true
+    source_code: 'https://vetiver.rstudio.com'
+    display_pins: true
+params:
+    board: !r pins::board_rsconnect()
+    name: 'julia.silge/hotel_rf'
+    version: NULL
+---
+
+```{r setup, include = FALSE}
+library(vetiver)
+library(tidyverse)
+library(tidymodels)
+library(pins)
+library(plotly)
+knitr::opts_chunk$set(echo = FALSE)
+
+## load new validation data, for example from database or API
+validation_data_url <- 'https://tidymodels.org/start/case-study/hotels.csv'
+validation_df <- read_csv(validation_data_url) %>%
+    mutate_if(is.character, as.factor) %>%
+    arrange(arrival_date) %>%
+    filter(arrival_date > "2017-01-01") 
+
+v <- vetiver_pin_read(params$board, params$name, version = params$version)
+v_meta <- pin_meta(params$board, params$name)
```

Comment:
This line is here because the next step will involve getting the metrics from training time out from the metadata.